Jenkinsfile

```diff
@@ -71,62 +71,62 @@ node (node_label) {
 
 	// stage to run http scale test
 	if (http == "TRUE") {
-		load "scale-ci/pipeline-scripts/http.groovy"
+		load "pipeline-scripts/http.groovy"
 	}
 
 	// stage to run deployments per ns test
 	if ( deployments_per_ns == "TRUE" ) {
-		load "scale-ci/pipeline-scripts/deployments_per_ns.groovy"
+		load "pipeline-scripts/deployments_per_ns.groovy"
 	}
 
 	// stage to run podvertical test
 	if ( podvertical == "TRUE" ) {
-		load "scale-ci/pipeline-scripts/podvertical.groovy"
+		load "pipeline-scripts/podvertical.groovy"
 	}
 
 	// stage to run networking test
 	if ( networking == "TRUE" ) {
-		load "scale-ci/pipeline-scripts/networking.groovy"
+		load "pipeline-scripts/networking.groovy"
 	}
 
 	// stage to run pgbench scale test
 	if ( pgbench_test == "TRUE" ) {
-		load "scale-ci/pipeline-scripts/pgbench.groovy"
+		load "pipeline-scripts/pgbench.groovy"
 	}
 
 	// stage to run mongodb ycsb scale test
 	if ( mongodb_ycsb_test == "TRUE" ) {
-		load "scale-ci/pipeline-scripts/mongodbycsb.groovy"
+		load "pipeline-scripts/mongodbycsb.groovy"
 	}
 
 	// stage to run mastervertical scale test
 	if (mastervertical == "TRUE") {
-		load "scale-ci/pipeline-scripts/mastervertical.groovy"
+		load "pipeline-scripts/mastervertical.groovy"
 	}
 
 	// stage to run ns_per_cluster test
 	if ( ns_per_cluster == "TRUE" ) {
-		load "scale-ci/pipeline-scripts/ns_per_cluster.groovy"
+		load "pipeline-scripts/ns_per_cluster.groovy"
 	}
 
 	// stage to run logging scale test
 	if (logging == "TRUE") {
-		load "scale-ci/pipeline-scripts/logging.groovy"
+		load "pipeline-scripts/logging.groovy"
 	}
 
 	// stage to run prometheus scale test
 	if ( prometheus_test == "TRUE" ) {
-		load "scale-ci/pipeline-scripts/prometheus.groovy"
+		load "pipeline-scripts/prometheus.groovy"
 	}
 
 	// stage to run BYO scale test
 	if (byo == "TRUE") {
-		load "scale-ci/pipeline-scripts/byo.groovy"
+		load "pipeline-scripts/byo.groovy"
 	}
 
 	// stage to run OCP 4.X scaleup
-	if (ocp_4.x_scaleup == "TRUE") {
-		load "scale-ci/pipeline-scripts/OCP-4.X/scaleup.groovy"
+	if (ocp4_scaleup == "TRUE") {
+		load "pipeline-scripts/"
```

Comment:
Should this have a specific file it is linked to?

Reply:
It should point to the scaleup_4.x groovy script, fixed it. Thanks.